### ISSUE TITLE
WeNet RelPositionalMHA plugin can't add_plugin_v2

### ISSUE THREAD

**Issue by CUGfred** [NVIDIA/TensorRT#2209]
## Description
i tried to output a wenet relpositionalMHA onnx model and create an attention node instead of ops. after developed plugin and i just can't add_plugin_v2 which cause segmentation fault, while i can't debug

## Environment

**TensorRT Version**: 8.4.1.5
**NVIDIA GPU**: A100
**NVIDIA Driver Version**: 450.102
**CUDA Version**: 11.0
**CUDNN Version**: 
**Operating System**: ubuntu 2004
**Python Version (if applicable)**: 3.8
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.11
**Baremetal or Container (if so, version)**: Container

## Steps To Reproduce
i use torch 1.11 to init a relpositionalmultiheadattention model and export as a onnx model, which got five inputs(q.k.v.m.p), two output(otuput, new_cache)
[https://github.com/wenet-e2e/wenet/blob/main/wenet/transformer/attention.py](url)
then i remove all node and some initializer via onnx python, append a custom node input are(q, mask, posemb), make my plugin, parameter get into plugin via fileldcollection.
while i use network.add_plugin_v2, i get segmentation fault

**Comment by zerollzeng** [comment]
Can you share the reproduced sample so that we can reproduce the segmentation fault on our side? BTW I think it may cause by the plugin implementation, I would suggest debugging it using GDB first.

**Comment by CUGfred** [comment]
@zerollzeng  thanks for your reply.
i just use gdb, it stop at parser.parse(model.read()) which met segmentation fault,  it ran like below
-> if not parser.parse(model.read()):
(Pdb) 
[cpp]getPluginVersion begin.
[cpp]getPluginNamespace begin.
[cpp]getPluginNamespace begin.
[cpp]getFieldNames begin.
[cpp]createPlugin begin.
[cpp]plugin initialize done.
[cpp]getNbOutputs begin.
[cpp]getNbOutputs begin.
[cpp]clone begin.
[cpp]setPluginNamespace begin.
Segmentation fault (core dumped)

maybe plugin cause this error.
to reproduce this is a little complicated, i use wenet transformer/attention.py to output a simple relpositional multihead attention model, then use onnx-graphsurgeon to fuse all those op into one custom attention op.

**Comment by CUGfred** [comment]
sry, after debug, it caused by code function supportsFormatCombination didn't suit the custom op.
